lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -184,6 +184,41 @@ function gutenberg_get_global_settings( $path = array(), $context = array() ) {
 }
 
 /**
+ * Repeated logic from `get_default_block_editor_settings` function. When implemented into core,
+ * remove logic from `get_default_block_editor_settings` and simple call this function instead.
+ *
+ * @return array
+ */
+function gutenberg_get_block_theme_supports() {
+	$theme_settings = array(
+		'disableCustomColors'    => get_theme_support( 'disable-custom-colors' ),
+		'disableCustomFontSizes' => get_theme_support( 'disable-custom-font-sizes' ),
+		'disableCustomGradients' => get_theme_support( 'disable-custom-gradients' ),
+		'disableLayoutStyles'    => get_theme_support( 'disable-layout-styles' ),
+		'enableCustomLineHeight' => get_theme_support( 'custom-line-height' ),
+		'enableCustomSpacing'    => get_theme_support( 'custom-spacing' ),
+		'enableCustomUnits'      => get_theme_support( 'custom-units' ),
+	);
+
+	// Theme settings.
+	$color_palette = current( (array) get_theme_support( 'editor-color-palette' ) );
+	if ( false !== $color_palette ) {
+		$theme_settings['colors'] = $color_palette;
+	}
+
+	$font_sizes = current( (array) get_theme_support( 'editor-font-sizes' ) );
+	if ( false !== $font_sizes ) {
+		$theme_settings['fontSizes'] = $font_sizes;
+	}
+
+	$gradient_presets = current( (array) get_theme_support( 'editor-gradient-presets' ) );
+	if ( false !== $gradient_presets ) {
+		$theme_settings['gradients'] = $gradient_presets;
+	}
+
+	return $theme_settings;
+}
+
```

Comment:
```suggestion
/**
```

There's a broken comment and this code cannot be executed.